index.html

```diff
@@ -0,0 +1,162 @@
+              <a href="#program">Program</a>
+            </li>
+            <li class="big-menu-right">
+              <a href="#speak">Speakers</a>
+            </li>
+            <li class="big-menu-right">
+              <a href="#sponsors">Sponsors</a>
+            </li>
+            <li class="big-menu-right">
+              <a href="index.html">News</a>
+            </li>
+            <li class="big-menu-right"><button class="contact">Contact Us</button>
+            </li>
+          </ul>
+        </div>
+        </nav>
+    <!-- Top Menu Desktop-->
+    </header>
+    <!-- Main Welcome Text-->
+    <section>
+      <div>
+        <span id="greeting" class="greeting">"Hello! Energy Event!</span>
+        <h1 class="event-name">ENERGY EVENT<br>
+          GLOBALLY<br>
+          2024</h1>
+        <p class="event-description">
+          World event about Renewable Energy, Solar Energy, Clean Energy, with fantastic speakers and workshops. Many
+          new
+          technologies are coming to the world, and we are going to discuss how to use them. Best of all, we are going
+          to
+          be the best event in the world on these technologies.
+        </p>
+      </div>
+      <div class="date-event">
+        <h2 class="date">2024.07.12(FRI) ~ 13(SAT)</h2>
+        <p class="place">
+          @ Oscar Niemeyer Hall, Conference room, Goiânia<br>
+        </p>
+      </div>
+    </section>
+  </div>
+  <!-- Main Programma 5 Cards-->
+  <section class="program" id="program">
+    <header>
+      <h3>Main Program</h3>
+      <div class="space"></div>
+    </header>
+    <ul class="cards" id="cards-js">
+    </ul>
+    <a id="a" class="mobile-none cards-link" href="#">SEE THE WHOLE PROGRAM</a>
+    <button type="button" class="btn-join-us">Join Us Today
+    </button>
+  </section>
+  <!-- Featured Speakers 6 Mobile Only-->
+  <section class="cont-persons desktop-none">
+    <h2>Featured Speakers</h2>
+    <ul class="persons" id="persons-first"></ul>
+    <ul class="persons less" id="persons-more"></ul>
+    <div id="more-cnt" class="mobile">
+      <button type="button" class="myBtn" id="more-bttn">
+        <span id="more-txt">MORE</span><span><img src="assets/images/arrow-down.png" alt="arrowhead" width="30"
+            id="arrowhead" /></span>
+      </button>
+    </div>
+  </section>
+    <!-- Featured Speakers 6 Desktop Only-->
+    <section class="cont-persons-desktop mobile-none" id="speak">
+      <h2>Featured Speakers</h2>
+      <ul class="persons" id="persons-all"></ul>
+      
+    </section>
+  <!-- Partners -->
+  <section class="partners" id="sponsors">
+    <h2>Partners</h2>
+    <div class="partner-logos">
+      <img src="assets/images/logo1.png" alt="logo" />
+      <img src="assets/images/logo2.png" alt="logo2" />
+      <img src="assets/images/logo3.png" alt="logo3" />
+      <img src="assets/images/logo4.png" alt="logo4" />
+      <img src="assets/images/logo5.png" alt="logo5" />
+    </div>
+  </section>
```

Comment:
**Suggested Changes**
- [ ] The partner section doesn't resemble the given design. Let us please try and follow the given design. I have attached screenshots of the differences. You could use texts rather than images.

Your Design
![your-partner](https://user-images.githubusercontent.com/7220167/143535997-8c3332c4-7ab9-4764-9087-e40f4c10418d.png)

Given Design
![partners](https://user-images.githubusercontent.com/7220167/143536028-e625a920-1b59-4a44-8d92-ea1a15872c8b.png)

Reply:
Talked to the previous reviewer and accepted to use logos instead of text. 
Items are now in a single row, smaller, and look like the original design.